src/components/dashboard/Claim.js

```diff
@@ -239,6 +240,9 @@ const Claim = props => {
 
   const faceRecognition = () => {
     //await handleClaim()
+    //temporary solution in the zero phase, for the situation when the user is not in the whitelist.
+    showSupportDialog(showErrorDialog, hideDialog, screenProps.push)
+
     screenProps.push('FRIntro', { from: 'Claim' })
```

Comment:
1. when we're going to the FR if we don't have in on phase 0 ?
2. apply your temporary solution only if we're in phase 0 (use env variable we defined)